packages/react-components/react-alert/src/stories/AlertAction.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { DismissCircleRegular } from '@fluentui/react-icons';
+
+import { Alert } from '../index';
+
+export const Action = () => (
+  <div style={{ display: 'flex', flexDirection: 'column', gap: '10px' }}>
+    <Alert
+      intent="success"
+      action={{
+        appearance: 'transparent',
+        children: 'Undo',
+      }}
```

Comment:
Due `padding` on `Button` this area is clickable:

![image](https://user-images.githubusercontent.com/14183168/168792515-1024d9ce-c79e-466b-a775-1dff60a88fbd.png)

Probably it's better reset `padding` and use `margin` instead

Reply:
Good catch, updated